src/server/models/db/migrations/20220913152735_missing_countries_lawyers/migration.sql

```diff
@@ -0,0 +1,20 @@
+-- Adds all remaining countries for lawyers service
+
+INSERT INTO "List" ("reference", "updatedAt", type, "countryId", "jsonData")
+
+-- Add these values to the columns stated above
+
+SELECT gen_random_uuid(), NOW(), 'lawyers', missingCountries."id", '{"publishers": ["ali@cautionyourblast.com","tom.evans@fcdo.gov.uk"], "validators": ["ali@cautionyourblast.com","tom.evans@fcdo.gov.uk"], "administrators": ["ali@cautionyourblast.com","tom.evans@fcdo.gov.uk"]}'
```

Comment:
minor but could you change the email addresses to ali.salaman@fcdo.gov.uk, ros.vaughan@fcdo.gov.uk and tom.evans@fcdo.gov.uk so we can access them from FC in prod.

Reply:
Yeah sure, it shouldn't be too difficult

Reply:
ah sorry @RichardBray , Ros' email address is rosalyn.vaughan@fcdo.gov.uk.  Could you please make this change to all 3 migration scripts then this is the only remaining change and can be merged in.